backend/__tests__/controllers/users.spec.ts

```diff
@@ -59,7 +59,9 @@ describe('/user routes', () => {
       });
   });
 
-  test('GET request with no id returns all users', (done) => {
+  // Test is being skipped because functionality was changed
+  // Now `get` only returns one user based on ext_it
+  test.skip('GET request with no id returns all users', (done) => {
```

Comment:
Is there a test that covers this functionality or should we adapt this test?

Reply:
I think we should just adapt this test; I know we were talking about some problems regarding the token getting passed properly (which affected this test), but I think we might have fixed it and might be able to test it now.